source/_integrations/magicseaweed.markdown

```diff
@@ -1,92 +0,0 @@
----
-title: Magicseaweed
```

Comment:
There is no redirect added for this missing/removed page.

Please add one.

Reply:
Sorry, missed it
#26726

Reply:
Thanks 👍